$OpenBSD: patch-src_GeoMath_cpp,v 1.1 2019/03/18 13:09:28 cwen Exp $

ports-gcc fix for "error: 'isnan' was not declared in this scope"

Index: src/GeoMath.cpp
--- src/GeoMath.cpp.orig
+++ src/GeoMath.cpp
@@ -26,7 +26,12 @@
 #include <tzdata.h>
 #endif
 
+#if defined(__OpenBSD__) && !defined(__clang__)
+#include <cmath>
+#define isnan std::isnan
+#else
 #include <math.h>
+#endif
 
 #if WIN32
 #include <float.h>
